Jeszcze jedno - live wyniki (#2581)
tpwres/teczki-polskiego-wrestlingu: content/e/ppw/2026-04-11-ppw-jeszcze-jedno.md

* On 4.04.2026 PpW announced [Vic Golden](@/w/vic-golden.md) coming back to straighten out the problems within [Rodzina](@/tt/rodzina.md).
* Two days later, [Max Speed](@/w/max-speed.md) vs ["Ladykiller" Boro](@/w/boro.md) was announced, pitting the young blood on his way towards a championship against the veteran.

## Card 

> {% card() %}

Reviewer: ```suggestion
{% card() %}
- - '[Gabriel Queen](@/w/gabriel-queen.md)'
  - '[Sambor](@/w/sambor.md)
  - s: Pre-show match
```

- - '[Max Speed](@/w/max-speed.md)'
  - '["Ladykiller" Boro](@/w/boro.md)'
- - '[Max Speed](@/w/max-speed.md)'
  - '["Ladykiller" Boro](@/w/boro.md)'
- - 'Bad Trip: [Goblin](@/w/goblin.md) & [Gustav Gryffin](@/w/gustav-gryffin.md) + [Biesiad Strong](@/w/biesiad.md)'
  - '[Zmowa](@/tt/zmowa.md): [Jakub](@/w/jakub.md) & [Mister Z](@/w/mister-z.md) & [Olgierd](@/w/olgierd.md)'
  - s: Anarchy Cyclone Tag Team Match
- credits:
    Host, Ring Announcer: '["Real OG" Michael HT](@/w/michael-ht.md)'
    Referees: '[Sędzia Kornel](@/w/sedzia-kornel.md), [Sędzia Seweryn](@/w/sedzia-seweryn.md)'
    Commentary: '[Istociarz](@/w/istociarz.md)'
{% end %}

### Recap

* Present in the chat during the live stream was somebody with the screen name "Piekny Kawaler". Whether or not this was the actual [Piękny Kawaler](@/w/piekny-kawaler.md), is unknown.
* [Oskar Alexander](@/w/oskar-alexander.md) was booked to fight [Jacob Crane](@/w/jacob-crane.md) but was replaced by [Filip Fux](@/w/filip-fux.md), owing to Oskar's apparent injury.
* Oskar Alexander attemted to help Filip Fux by handing him his crutch, but he was stopped by Gabriel Queen and Agentka Agatka, despite being seemingly on the same side.

> * [Oskar Alexander](@/w/oskar-alexander.md) and [Filip Fux](@/w/filip-fux.md) swore loyalty to [Vic Golden](@/w/vic-golden.md) and [Rodzina](@/tt/rodzina.md), as did [Robert Star](@/w/robert-star.md) via a pre-recorded voice message. [Agentka Agatka](@/w/agentka-agatka.md) refused, stating that being there for the family is more important than any oaths, and Vic wasn't there when needed. When it came to Queen, Vic demanded he knelt - which Queen did, only to insult and attack him. Moments later, Agatka attacked Filip - thus her and Queen broke off from Rodzina.

Reviewer: ```suggestion
* [Oskar Alexander](@/w/oskar-alexander.md) and [Filip Fux](@/w/filip-fux.md) swore loyalty to [Vic Golden](@/w/vic-golden.md) and [Rodzina](@/tt/rodzina.md), as did [Robert Star](@/w/robert-star.md) via a pre-recorded voice message. [Agentka Agatka](@/w/agentka-agatka.md) refused, stating that being there for the family is more important than any oaths, and Vic wasn't there when needed. When it came to Queen, Vic demanded he knelt - which Queen did, only to insult and attack him. Moments later, Agatka attacked Filip - thus her and Queen broke off from Rodzina, to a monstrous pop from the audience.
```

* [Max Speed](@/w/max-speed.md) pinned [Boro](@/w/boro.md) at the first attempt, after only a few seconds into the match. Boro then asked for "one more" match, claiming that he hadn't been ready. Max agreed, and won again.
* During the main event [Gustav Gryffin](@/w/gustav-gryffin.md) and [Biesiad Strong](@/w/biesiad.md) argued about the use of tables. They decided to resolve the situation with a quick game of rock-paper-scissors, only for both of them to throw middle fingers.
* According to the commentary, some of the wrestlers continued to fight backstage.

> * Following the main event, Michael HT mentioned "one more" thing - a video of MBM, last seen at [Miasto Bezprawia](@/e/ppw/2024-02-10-ppw-miasto-bezprawia.md), saying he's going to appear at Ledwo Legalne 6.

Reviewer: ```suggestion
* Following the main event, Michael HT mentioned "one more" thing - a video of Belgian wrestler MBM, last seen at [Miasto Bezprawia](@/e/ppw/2024-02-10-ppw-miasto-bezprawia.md), announcing his appearance at Ledwo Legalne 6.
```

* On 4.04.2026 PpW announced [Vic Golden](@/w/vic-golden.md) coming back to straighten out the problems within [Rodzina](@/tt/rodzina.md).
* Two days later, [Max Speed](@/w/max-speed.md) vs ["Ladykiller" Boro](@/w/boro.md) was announced, pitting the young blood on his way towards a championship against the veteran.

## Card 

{% card() %}
- - '[Gabriel Queen](@/w/gabriel-queen.md)'

>   - '[Sambor](@/w/sambor.md)

Reviewer: ```suggestion
  - '[Sambor](@/w/sambor.md)'
```
